machine_management/adding-rhel-compute.adoc

```diff
@@ -0,0 +1,30 @@
+[id="adding-rhel-compute"]
+= Adding RHEL compute machines to a {product-title} cluster
+include::modules/common-attributes.adoc[]
+:context: adding-rhel-compute
+
+toc::[]
+
+In {product-title}, you can add Red Hat Enterprise Linux (RHEL) compute, or
+worker, machines to a user provisioned infrastructure cluster. You can use RHEL
+as the operating system on only compute machines.
+
+include::modules/rhel-compute-overview.adoc[leveloffset=+1]
+
+include::modules/rhel-compute-requirements.adoc[leveloffset=+1]
+
+include::modules/rhel-preparing-playbook-machine.adoc[leveloffset=+1]
+
+include::modules/rhel-preparing-node.adoc[leveloffset=+1]
+
+include::modules/rhel-adding-node.adoc[leveloffset=+1]
+
+include::modules/rhel-ansible-parameters.adoc[leveloffset=+1]
+
+[id="adding-rhel-compute-removing-rhcos"]
+== (Optional) Removing RHCOS compute machines from your cluster
+
+After you add the RHEL compute machines to your cluster, you can remove the
+RHCOS compute machines.
+
+include::modules/rhel-removing-rhcos.adoc[leveloffset=+2]
```

Comment:
"Removing RHCOS compute machines from your cluster" showed twice in my web browser, at top of page
(Optional) Removing RHCOS compute machines from your cluster
    Removing RHCOS compute machines from a cluster